packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -242,6 +279,9 @@ try {
 					});
 				}
 			},
+			afterWordPressInstall(php) {
```

Comment:
You are correct. I got to excited about boot hooks and wanted to use them 😄 
Removed it in [8beee0c](https://github.com/WordPress/wordpress-playground/pull/1532/commits/8beee0c1e99d5b4f6e2c7e48faae56ddbc93ebcb)

Reply:
> You are correct. I got to excited about boot hooks and wanted to use them 😄

Me too. I grinned when first reading this PR because I had been doing the same things on Friday. It felt like we were on the same page by accident.